### PR TITLE
[W5.4][T15-A2] Aw Wen Hao

### PR DESCRIPTION
**Pull request by 17navasaw**

### CODE REVIEW

**Comment by aziziazfar** [comment]
Well done! Code was well written and easy to understand

File: src/seedu/addressbook/commands/DeleteRangeCommand.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    public DeleteRangeCommand(ArrayList<Integer> targetVisibleIndices) {
+        this.targetVisibleIndices = targetVisibleIndices;
+    }
+
+    @Override
+    public CommandResult execute() {
+        int startIndex = targetVisibleIndices.get(0);
+        int endIndex = targetVisibleIndices.get(1);
+
+        for (int i=startIndex;i<=endIndex;i++) {
```
Comment (K1ang) [inline src/seedu/addressbook/commands/DeleteRangeCommand.java]: You can make this more readable by following the java convention for `for` loops.

File: src/seedu/addressbook/data/person/Address.java [hunk 2]
```diff
@@ -23,17 +34,47 @@
...
+                    postalCode = new PostalCode(separator.nextToken());
+                    break;
+                default:
+                    return count;
+            }
+        }
+
+        return count;
+    }*/
+
```
Comment (K1ang) [inline src/seedu/addressbook/data/person/Address.java]: Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.

File: src/seedu/addressbook/data/person/Street.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package seedu.addressbook.data.person;
+
+public class Street {
```
Comment (K1ang) [inline src/seedu/addressbook/data/person/Street.java]: Remember to include header comments for classes and non trivial methods.

**Comment by K1ang** [comment]
Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**